robocopkaka/event-manager-rails
# Ft users can add events ui 163861559

#### What does this PR do?
Adds form for creating events
#### Description of Task to be completed?
* Adds presentational and container components needed to create events
* Adds actions, action creators, and reducers needed to create events
* Adds a button to the CenterPage component for adding new events
#### How should this be manually tested?
* Visit `${baseUrl}/centers/:id`
* Click on the `Add Event` button on the center's page
* You should be redirected to a page with a form for adding events
* Fill in the details of the events
* If successful, you should see a `green` toast and be redirected back to the center's page
* If unsuccessful, you should see a `red` toast with the error messages
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? (if applicable)
[#163861559](https://www.pivotaltracker.com/story/show/163861559)
### Screenshots
![image](https://user-images.githubusercontent.com/11176000/60453994-99899180-9c2a-11e9-98e4-ba4f28038e50.png)
![image](https://user-images.githubusercontent.com/11176000/60454014-a8704400-9c2a-11e9-8af6-206a2e474036.png)
